pabi_hr_expense_report/pabi_dunning_letter/wizard/pabi_dunning_letter_wizard.py

```diff
@@ -124,3 +168,6 @@ class DunningList(models.TransientModel):
         related='expense_id.name',
         readonly=True,
     )
+    empty_email = fields.Boolean(
+        string="Email Empty?"
```

Comment:
Do we need to check for empty email for this employee? Because, even employee do not have email, we might send to other CC mail or group mail anyway. (I think we don't need it, just skip record if there is no email at all, WDYT?)